src/pages/Home/Home.jsx

```diff
@@ -1,19 +1,26 @@
 import React from "react";
 import { FormattedMessage } from "react-intl";
-import { Grid } from "@mui/material";
+import { Grid, Typography, Box } from "@mui/material";
 import CountdownTimer from "../../components/CountdownTimer/CountdownTimer";
```

Comment:
:question: Can `CountdownTimer` be imported like this...
`import CountdownTimer from "../../components/CountdownTimer";`

Reply:
Addressed:  54225a2b77422a74c436c9fcd4dddbf8ea2c3606